README.md

```diff
@@ -132,7 +132,6 @@ To add a regular project, create a new markdown file in the `_projects` folder w
 ```yml
 layout: post
 title: "Cats"
-category: "Functional Programming"
```

Comment:
Add the platforms line as part of the example?

I can't add it as a suggestion :( 
`platforms: [js, jvm, native]`

Reply:
Oh closer inspection, this whole section might need a bit of a revamp. Line 128 is still referring to incubator projects, and the example also includes a `layout` key that I don't think is necessary.

Reply:
I updated the readme in [54fcf5c](https://github.com/typelevel/typelevel.github.com/pull/422/commits/54fcf5cd2f2bf92ed89663652221ca9507a6f78c)